### PR TITLE
Load fixers from StyleCI config instead of PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Note that the fixer names match that of 2.0, not 1.x. There are some differences.

**Comment by GrahamCampbell** [comment]
For example, phpdoc_align is named phpdoc_params in 1.x.

**Comment by GrahamCampbell** [comment]
Oh, yeh, I see you copied the aliases stuff over. :)

**Comment by GrahamCampbell** [comment]
You'll need to work the aliases backwards to be compatible with 1.x.

File: .styleci.yml [hunk 1]
```diff
@@ -12,3 +12,7 @@ disabled:
   - psr0
   - unalign_double_arrow
   - unalign_equals
+
+finder:
+  not-name:
+    - "*.php.twig"
```
Comment (GrahamCampbell) [inline .styleci.yml]: Shouldn't that be excluded by default anyway, because it doesn't match "*.php", it's "*.php*".

Comment (Soullivaneuh) [inline .styleci.yml]: That was I thinking. But no, php-cs-fixer with default config was trying to fix my `.php.twig`...

Comment (GrahamCampbell) [inline .styleci.yml]: Hmmm. Ok then. :)

**Comment by Soullivaneuh** [comment]
> Note that the fixer names match that of 2.0, not 1.x. There are some differences.

For aliases and missing fixers, this is plan on #10 and #13.

I just made a quick fix to make this bridge more compliant with StyleCI.

And yeah, we have Laravel preset now. \o/ ;-)

**Comment by GrahamCampbell** [comment]
:)

**Comment by GrahamCampbell** [comment]
Can I chat to you on slack or something?

**Comment by GrahamCampbell** [comment]
Or skype. Got some things I want to discuss with you. :)

**Comment by Soullivaneuh** [comment]
The best for me is to contact me by mail at `soullivaneuh(at)gmail.com`. :+1: 

You can also add me at hangout if you want direct chat.

**Pull request merged**